plyer/platforms/android/contacts.py

```diff
@@ -0,0 +1,67 @@
+"""This module provides access for contact list."""
+
+from plyer.facades import Contacts
+from plyer.platforms.android import activity
+
+from jnius import autoclass
+
+JavaContacts = autoclass('android.provider.ContactsContract$Contacts')
+Phone = autoclass('android.provider.ContactsContract$CommonDataKinds$Phone')
+ArrayList = autoclass('java.util.ArrayList')
+String = autoclass('java/lang/String')
+
+
+class AndroidContacts(Contacts):
+
+    """Android Contacts.
+
+    .. versionadded:: 1.2.4
+
+    """
+
+    def refresh(self):
+        """Refresh local contact list."""
+        cr = activity.getContentResolver()
+        contact_cr = cr.query(JavaContacts.CONTENT_URI, None, None, None, None)
+        if contact_cr.getCount < 1:
+            return
```

Comment:
getCount is not a method? Also if there is no contact, you should reset the _contacts as well